wgpu-core/src/instance.rs

```diff
@@ -89,16 +89,29 @@ pub struct Instance {
     supported_backends: Backends,
 
     pub flags: wgt::InstanceFlags,
+
+    /// Non-lifetimed [`raw_window_handle::DisplayHandle`], only for validation purposes
+    /// in [`Self::create_surface()`].
+    display: Option<raw_window_handle::RawDisplayHandle>,
 }
 
+// HACK because of storing RawDisplayHandle for validation purposes
```

Comment:
```suggestion
// We store the RawDisplayHandle for validation purposes, never accessing the underlying handles.
```

Not really a hack

Reply:
I'm quite convinced that forcing `RawDisplayHandle` to be `Send`/`Sync` when `raw_window_handle` explicitly makes (and tests) that they're not `Send`/`Sync` is invalid.

For now I've rewrapped these, but it's all perpendicular to your latest request to using `Instance` keep a `dyn HasDisplayHandle` alive as long as it's `Send`/`Sync`.